openbb_terminal/parent_classes.py

```diff
@@ -1181,3 +1192,9 @@ def call_load(self, other_args):
                     ns_parser.exchange,
                     self.current_interval,
                 )
+                export_data(
+                    ns_parser.export,
+                    os.path.dirname(os.path.abspath(__file__)),
+                    "load",
+                    self.current_df.copy(),
```

Comment:
Why `copy()` here?

Reply:
i don't want to goof up the dataframe lol

Reply:
What do you mean by goof up? At that stage we just save it, there's nothing happening with it after.

Reply:
I had an edge case in my brain at the time but can't think of what it is now